src/styles/styles.js

```diff
@@ -1547,8 +1547,8 @@ const styles = {
             // paddingVertical: 0, alignSelf: 'center', and textAlignVertical: 'center'
 
             paddingHorizontal: variables.avatarChatSpacing,
-            paddingTop: 0,
-            paddingBottom: 0,
+            paddingTop: 5,
```

Comment:
@dukenv0307 Your proposal mentioned that if the numberOfLines would be 1, the padding values should be 9. I don't see that change here, can you please explain?

Reply:
I think that's covered in the `textInputCompose` style right above this? The styles will be used conditionally: `this.props.numberOfLines > 1 ? styles.textInputComposeMultiLines : styles.textInputCompose,`

Reply:
I just updated